packages/channel-provider/src/logger.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
Would it make sense to disable this rule per-line?

Reply:
Maybe, but it made it less readable.